op-chain-ops/genesis/config.go

```diff
@@ -912,15 +934,24 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 		"_initializing": false,
 		"bridge":        predeploys.L2StandardBridgeAddr,
 	}
+
+	excessBlobGas := block.ExcessBlobGas()
+	if excessBlobGas == nil {
+		excessBlobGas = u64ptr(0)
+	}
```

Comment:
Was confused if this was the L2 or L1 block. Updating the doc comment on the function to specify it is a L1 block (anchor of the chain) would be helpful.